changedetectionio/content_fetcher.py

```diff
@@ -5,6 +5,7 @@
 from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
 from selenium.webdriver.common.proxy import Proxy as SeleniumProxy
 from selenium.common.exceptions import WebDriverException
+from playwright.sync_api import sync_playwright
```

Comment:
We've already included the dependency in requirements.txt, so I don't think importing a module would cause an exception. Wasn't Selenium also imported at the top of this file?

Reply:
You can import whatever you like in `requiremens.txt` but doesnt mean it's going to run the code

> Wasn't Selenium also imported at the top of this file?

yes but you can change that too please :)

Reply:
> You can import whatever you like in `requiremens.txt` but doesnt mean it's going to run the code

Doesn't one need to run `pip install -r requirements.txt` before developing or using this software?

Reply:
sure but please just move that import :) and the selenium one :)